propel/resource_data_pool_access_policy_test.go

```diff
@@ -14,6 +15,10 @@ import (
 func TestAccPropelDataPoolAccessPolicyBasic(t *testing.T) {
 	ctx := map[string]any{}
 
+	if v := os.Getenv("PROPEL_CLIENT_ID"); v != "" {
+		ctx["app_id"] = v
+	}
```

Comment:
yes, actually, the test previously was using a placed application id so that was failing when you run it in another place diff that GitHub pipeline. With the change now it uses the actual configuration that is pass through env variables